lib/__tests__/browser/usersVisibility.js

```diff
@@ -0,0 +1,67 @@
+import cheerio from 'cheerio';
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import should from 'should/as-function';
+
+import RenderServer from '../fixtures/RenderServer';
+import ApiServer from '../fixtures/ApiServer';
+
+const { load } = cheerio;
+const TIME_OUT = 250;
+
+function removeReactAttributes(html) {
+  return html.replace(/ (data-reactid|data-react-checksum)=".*?"/g, '');
+}
+
+const { after, before, describe, it, browser } = global;
+describe('[FT] Users Visibility', () => {
+
+  let renderServer = null;
+  let apiServer = null;
+
+  before(async function $before(done) {
+    apiServer = new ApiServer({ port: 0 });

```

Comment:
It would be interesting to put this code somewhere and reuse it in each test file. Same for `after`.